src/SupervisionSuccess/model/calcOutcomesProportions.js

```diff
@@ -0,0 +1,59 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+
+/**
+ * Function that calculates outcome proportions from the given inputs.
+ * @param {number} changeInRevocations - integer value of the percentage change in revocations
+ * @param {number} changeInNewAdmissions - integer value of the percentage change in new admissions
+ * @const {number} prisonPopulationDiff - total difference of people in prison
+ * @returns {number}
+ */
+function calcOutcomesProportions(
+  finalRevocations,
+  finalAdmissions,
+  changeInRevocations,
+  changeInNewAdmissions,
+  prisonPopulationDiff,
+  baseRevocations,
+  baseAdmissions
+) {
```

Comment:
I think there's a bug in the logic somewhere. In this example, we see that there is a slight decrease in revocations and a slight increase in new admissions, but the latter is larger (in absolute terms) than the former, so we should expect it to say 100% for new admissions and say nothing for revocations. Instead, it says 100% for revocations.

<img width="1289" alt="Screen Shot 2021-07-16 at 11 21 43 AM" src="https://user-images.githubusercontent.com/431895/125986168-3959611a-a072-4671-87c0-fe2f7ac31417.png">

Reply:
I think it _might_ be because of the if block at the end, i.e. `else if (Math.abs(changeInRevocations) > Math.abs(changeInNewAdmissions))`. In that case, you're simply comparing the percentage changes themselves, but what really matters is the absolute change (in the case above, 1% of new admissions is quite a bit larger than 3% of revocations).

Reply:
Yeah, I see now, fixed.

Reply:
@nasaownsky it doesn't look like you've pushed up a new commit, so I can't verify the fix just yet. Is that coming soon?

Reply:
Thanks for the new commit! Will be able to verify this after the other issue I called out above is fixed.